terragrunt/aws/s3/web.tf

```diff
@@ -0,0 +1,43 @@
+locals {
+  s3_origin_id = "upd-web-${var.env}"
+}
+
+resource "aws_s3_bucket" "cra_upd_web_bucket" {
```

Comment:
FYI - we do have terraform module for buckets that you can use that will set the default options we require at CDS (such as not making them publicly accessible) - https://github.com/cds-snc/terraform-modules/tree/main/S3